README.md

```diff
@@ -1,14 +1,12 @@
 # Deep Teaching
 
-You've put deep learning on your todo list, but how about deep teaching? This one day workshop introduces education best practices to programmers and data scientists.
+You've put deep learning on your to-do list, but how about deep teaching? This one day workshop introduces education best practices to programmers and data scientists.  Among the topics it covers are:
 
-If you weren't able to attend our instructor training workshop at rstudio::conf, and would like to learn more about effective teaching, we have good news: the Education team is offering the workshop online for RStudio staff in March.  Among the topics we will cover are:
-
-- how to figure out who your learners actually are,
-- how the differences between novices, competent practitioners, and experts affect what and how you teach,
-- using formative assessment to keep teaching on track in real time,
-- what cognitive load is and how to design lessons that take it into account,
-- teaching as a performance art, and
+- how to figure out who your learners actually are;
+- how the differences between novices, competent practitioners, and experts affect what and how you teach;
+- using formative assessment to keep teaching on track in real time;
+- what cognitive load is and how to design lessons that take it into account;
+- teaching as a performance art; and
 - what motivates and demotivates adult learners.
 
-This workshop will be very interactive; in order to accommodate everyone's schedule, we will offer the training at an American-friendly time on Tuesday March 19 and at a European-friendly time on Tuesday March 26. If you are interested, please go to https://goo.gl/forms/EFmiq5Kv4sURayMe2 and let us know what time(s) work best for you - we'll schedule it to fit as many of your calendars as we can. If you have any questions, please Slack or email Greg Wilson.
+For more information, please contact [Garrett Grolemund](garrett@rstudio.com) or [Greg Wilson](greg.wilson@rstudio.com).
```

Comment:
As lead instructor, you should probably be first contact ;)

Reply:
Alpha by surname for all collaborative works.

Reply:
Sure, for equal collaborations; otherwise we should devolve to the practice of First Author. Please feel free to change this if you ever like. For now I'll merge it as is.